src/app/tests/suites/certification/Test_TC_DRYERCTRL_2_1.yaml

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 3.2.1. [TC-DRYERCTRL-2.1]  Optional Dryness attributes with server as DUT
+
+PICS:
+    - DRYERCTRL.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Laundry Dryer Controls"
+    endpoint: 1
+
+    unsupportedSelectedDrynessLevel:
```

Comment:
I know this is more of a test plan thing, but is the device guaranteed to HAVE an unsupported selected dryness level?

Reply:
It doesn't have an unsupported dryness level it is just for the test case

Just to check if it doesn't accept an unsupported dryness level